classes/abstracts/ActionScheduler_Store.php

```diff
@@ -448,11 +448,15 @@ public function get_status_labels() {
 	 * @return string
 	 */
 	public function has_pending_actions_due() {
-		$pending_actions = $this->query_actions( array(
-			'date'    => as_get_datetime_object(),
-			'status'  => self::STATUS_PENDING,
-			'orderby' => 'none',
-		) );
+		$pending_actions = $this->query_actions(
+			array(
+				'per_page' => 1,
+				'date'     => as_get_datetime_object(),
+				'status'   => self::STATUS_PENDING,
+				'orderby'  => 'none',
+			),
+			'count'
+		);
 
 		return ! empty( $pending_actions );
```

Comment:
Even though this still works, it might be worth updating since `query_actions` returns a numeric string now that it's set to `count`, instead of an array. Maybe something like

```
	return absint( $pending_actions ) > 0;
```

What do you think?

Reply:
I think it's developer preference. Both approaches amount to the same thing: `absint()` will return `0` on anything that `empty()` would return as `true`, so doesn't really matter.